### PR TITLE
doc:add backup dir and add backup introduce

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
part of #404 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
doc:add backup dir and add backup introduce
```

### CODE REVIEW

File: docs/content/en/docs/fleet-manager/backup/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Unified Backup, Restore, and Migrate"
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/_index.md:2]: mIgration, it shoud not be verbs

File: docs/content/en/docs/fleet-manager/backup/_index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+linkTitle: "Unified Backup"
+weight: 40
+description: >
+  The easiest way to manage multi cluster Unified Backup, Restore, and Migration with Fleet.
+---
+
+Kurator introduces a unified solution for backup, restore, and migration across multiple clusters in Fleet.
+This feature, integrated with [Velero](https://velero.io/), simplifies the process of managing backups, restoring data, and migrating resources across clusters through a streamlined one-click operation.
+
+The following are the main benefits of this feature.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/_index.md:12]: Prefer introduce application backup, migration below other than just cluster resource

File: docs/content/en/docs/fleet-manager/backup/_index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Below is the architecture diagram of the Kurator Fleet Backup feature.
+
+{{< image width="100%"
+link="./image/backup-arch.svg"
+>}}
+
+The diagram illustrates:
+
+- **Fleet Manager**: The central management entity, responsible for orchestrating the operations.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/_index.md:34]:
```suggestion
- **Fleet Manager**: The core management component, responsible for managing fleet and backup engines.
```

File: docs/content/en/docs/fleet-manager/backup/_index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- **Velero Integration**: How the backup and restore processes are handled using Velero.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/_index.md:35]: There is no velero in the graph

Comment (hzxuzhonghu) [reply docs/content/en/docs/fleet-manager/backup/_index.md:35]: So we need to change a wording

File: docs/content/en/docs/fleet-manager/backup/_index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- **Object Storage**: The external storage solutions, such as Minio or cloud providers, where the backups are stored.
+- **Interactions**: The arrows depict the flow of data and interactions between various components.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/_index.md:37]: This is meaningless, i guess you want to elaborate how they interract, more clearly, the workflow.

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 1 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:2]: ok

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 2 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:12]: ok

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 3 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:34]: ok

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 4 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:35]: yes

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 5 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:37]: ok

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 4 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:35]: we can just remove it

File: docs/content/en/docs/fleet-manager/backup/_index.md (hunk 5 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/_index.md:37]: we can just remove it

**Comment by Xieql** [comment]
@hzxuzhonghu PTAL

File: docs/content/en/docs/fleet-manager/backup/backup-plugin.md [hunk 6]
```diff
@@ -1,30 +1,13 @@
 ---
 title: "Kurator Fleet Backup Plugin Configuration Guide"
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup/backup-plugin.md:2]: how about `install backup plugin`

Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup/backup-plugin.md:2]: ok

**Pull request merged**